misk/src/main/kotlin/misk/web/interceptors/RequestLoggingInterceptor.kt

```diff
@@ -122,7 +122,11 @@ class RequestLoggingInterceptor internal constructor(
     val sampling = if (isError) errorBodySampling else bodySampling
     val randomDouble = random.current().nextDouble()
     if (randomDouble < sampling) {
-      val requestResponseBody = bodyCapture.get()
+      val requestResponseBody = try {
+        bodyCapture.get()
+      } catch (e: Throwable) {
```

Comment:
I think it would be a lot more obvious to do an isPresent() check rather than catching an NPE.

Reply:
+1, exception based control is definitely less preferable if there's an available method we can use instead

Reply:
I've changed the code to use safe calls instead, please review again.